oak_functions/loader/src/main.rs

```diff
@@ -141,19 +172,34 @@ async fn async_main(opt: Opt, config: Config, logger: Logger) -> anyhow::Result<
         .and_then(|policy| policy.validate())?;
 
     let address = SocketAddr::from((Ipv6Addr::UNSPECIFIED, opt.http_listen_port));
-    // Start HTTP server.
-    let server_handle = tokio::spawn(async move {
-        create_and_start_server(
-            &address,
-            &wasm_module_bytes,
-            lookup_data,
-            config.policy.unwrap(),
-            async { notify_receiver.await.unwrap() },
-            logger,
-        )
-        .await
-        .context("error while waiting for the server to terminate")
-    });
+
+    // Start server.
+    let server_handle = match opt.protocol {
```

Comment:
If anything, I think this switch should be a config / feature. But, I think it's probably fine to just move over to gRPC completely already.